integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataRecordCursor.java

```diff
@@ -78,6 +78,10 @@ public CarbondataRecordCursor(CarbonReadSupport<Object[]> readSupport,
     return columnHandles.get(field).getColumnType();
   }
 
+  /**
+   * get next Row/Page
+   * @return
+     */
```

Comment:
please change format the comment as below : 
/**
 * get next Row/Page
 * @return
 */